src/server/game/Entities/Unit/Unit.cpp

```diff
@@ -11351,6 +11333,40 @@ uint32 Unit::SpellDamageBonusTaken(Unit* caster, SpellInfo const* spellProto, ui
     return uint32(std::max(tmpDamage, 0.0f));
 }
 
+float Unit::processDummyAuras(float TakenTotalMod) const {
+    // note: old code coming from TC, just extracted here to remove the code duplication + solve potential crash
+    // see: https://github.com/TrinityCore/TrinityCore/commit/c85710e148d75450baedf6632b9ca6fd40b4148e
```

Comment:
Unnecessary

Reply:
it is necessary so people don't assume this code is newly added when checking the code with annotations open (git blame)